code-analysis/crates/solidity/outputs/rust/tests/src/cst_output/generated/mod.rs

```diff
@@ -21,6 +22,7 @@ pub fn get_parser<'a>(
     return Ok(match parser_name {
```

Comment:
We should generate this method with all the parsers in it.

Reply:
Sounds good to me, and we can switch to that.